runconfig/parse.go

```diff
@@ -70,6 +72,8 @@ func Parse(cmd *flag.FlagSet, args []string) (*Config, *HostConfig, *flag.FlagSe
 	cmd.Var(&flVolumes, []string{"v", "-volume"}, "Bind mount a volume")
 	cmd.Var(&flLinks, []string{"#link", "-link"}, "Add link to another container")
 	cmd.Var(&flDevices, []string{"-device"}, "Add a host device to the container")
+	cmd.Var(&flLabels, []string{"l", "-label"}, "Set meta data on a container")
```

Comment:
I guess we could presume that it's attaching metadata to the container?

```
-l, -label    Set metadata (format: key[=value])
```

Or maybe assume the jargon, in the same way understanding "volume" and "link" is assumed?

```
-l, -label    Set label (format: key[=value])
```